src/NUnit3TestAdapterInstall/source.extension.vsixmanifest

```diff
@@ -12,9 +12,9 @@ For running NUnit 2 tests, use the NUnit 2 adapter.</Description>
         <Tags>test, Testing, adapter, NUnit, TDD, NUnit3, UnitTesting</Tags>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[11.0,16.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0,16.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[11.0,16.0)"/>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[11.0,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[11.0,17.0)" />
```

Comment:
I thought Visual Studio 2019 was dropping support for VSIX based test adapters. Am I mistaken?

Reply:
VS 2019 still supports vsix, and even if it is on its way to be deprecated, it seems it will still take some time.  As long as VS supports vsix I think we should just continue supporting it.  It is pretty easy to do, ... so far....